### PR TITLE
Reuse GetClusterRegInfo from mc

### PR DESCRIPTION
**Pull request by anjalshireesh**
The logic related to calculating no of pools has changed in mc, but still old in console. Ensuring that the function from mc is reused here will avoid such issues in future.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -155,3 +155,5 @@ require (
 	gopkg.in/ini.v1 v1.67.0 // indirect
 	gopkg.in/yaml.v3 v3.0.1 // indirect
 )
+
+replace github.com/minio/mc => github.com/anjalshireesh/mc v0.0.0-20240319134657-0e1592c105d9
```
Comment (bexsoft) [inline go.mod:159]: Please remove replace

**Review by bexsoft — CHANGES_REQUESTED** [review]

File: go.mod (hunk 1 above)
Comment (anjalshireesh) [reply go.mod:159]: @bexsoft will remove it after the corresponding PR https://github.com/minio/mc/pull/4881 is merged in `mc`.

Comment (anjalshireesh) [reply go.mod]: @bexsoft done, PTAL.

**Review by shtripat — APPROVED** [review]
lgtm

**Review by prakashsvmx — APPROVED** [review]

**Review by harshavardhana — APPROVED** [review]

**Pull request merged**